tutorials/preprocessing/plot_70_fnirs_processing.py

```diff
@@ -29,12 +29,35 @@
 raw_intensity = mne.io.read_raw_nirx(fnirs_raw_dir, verbose=True).load_data()
 
 
+###############################################################################
+# View location of sensors over brain surface
+# -------------------------------------------
+#
+# Here we validate that the location of sources-detector pairs and channels
+# are in the expected locations. Source-detector pairs are shown as lines
+# between the optodes, channels (the mid point of source-detector pairs) are
+# shown as dots.
+
+mne.datasets.fetch_fsaverage()
+subjects_dir = mne.datasets.sample.data_path() + '/subjects'
+mne.viz.set_3d_backend('pyvista')
+
+fig = mne.viz.create_3d_figure(size=(800, 600), bgcolor='white')
+fig = mne.viz.plot_alignment(raw_intensity.info, show_axes=True,
+                             subject='fsaverage',
+                             trans='fsaverage', surfaces=['brain'],
+                             fnirs=['channels', 'pairs'],
```

Comment:
You changed the default `True` to mean `('pairs',)` but then use `['channels', 'pairs']` in the example. Would `('channels', 'pairs')` might actually be the best default for `plot_alignment`? Or do you indeed want to do something non-default in this example? Sorry for the back and forth on this one...

Also would this look better with `surfaces=['brain', 'head']`? Might look cooler?

Reply:
I leave both channels and pairs in here as they are described above and I see this tutorial as a tool to show people the nirs processing options, then they can choose the option they prefer. I think pairs is still the best default.  Also I prefer brain alone without head here.